src/SeleniumLibrary/utils/events/event.py

```diff
@@ -15,9 +15,30 @@
 # limitations under the License.
 
 import abc
-
+from selenium.webdriver.support.event_firing_webdriver import EventFiringWebDriver
 
 class Event:
     @abc.abstractmethod
     def trigger(self, *args, **kwargs):
         pass
+
+def _unwrap_eventfiring_elements(ef_driver,elements):
+    """ Workaround for Selenium 3 bug.
+
+        References:
+        https://github.com/SeleniumHQ/selenium/issues/7877
+        https://github.com/SeleniumHQ/selenium/pull/8348
+        https://github.com/SeleniumHQ/selenium/issues/7467
+        https://github.com/SeleniumHQ/selenium/issues/6604
+
+    """
+    if not isinstance(ef_driver,EventFiringWebDriver) or selenium_major_version() >= 4:
+        return elements
+    unwrapped_elements = ef_driver._unwrap_element_args(elements)
```

Comment:
There is also [public method](https://selenium-python.readthedocs.io/api.html#selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement.wrapped_element) and same for the [driver](https://selenium-python.readthedocs.io/api.html#selenium.webdriver.support.event_firing_webdriver.EventFiringWebDriver.wrapped_driver). But do not know is there need to unwrap the driver.

Reply:
I came across that initially and then lost it till I saw the fix you had in there as I tested. One thing about this private method I liked was that it could handle a single element or a list of elements. I wasn't sure if any of the keywords I was patching had the possibility of multiple elements. Two other things I worried about. One was that there could be other reasons for an AttributeError exception (have no definitive reasons but also no way to prove that only this issue could cause one). The other concern was having too many try catch to pick up all the lines in the element.py file.

I'm not unwrapping the webdriver but just the webelements, using the private method on the EventFiringWebDriver. I see I could possibly just bring the element(s) and see if there instance of event firing webelement version. I guess I brought in the driver to have a way check the instance is a event firing web element.

Reply:
Usually action chains work with single element at the time. Therefore it would be safer to use the public attribute.

Reply:
Sounds good. I will go back and take a look at this.

Reply:
Switched over to the public attribute. Looking at the acceptance test now ..